kernelci.org/content/en/docs/instances/chromeos.md

```diff
@@ -0,0 +1,243 @@
+        flash_tarball_compression: 'gz'
+        tast_tarball: 'tast.tgz'
+      reference_kernel:
+        image: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-hatch/20220621.0/amd64/bzImage'
+        modules: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-hatch/20220621.0/amd64/modules.tar.xz'
+```
+* **filters**, indicates which kernel builds are suitable for this Chromebook, 
+fragment names can be found by the name of board in the ChromeOS sources.
+
+For example: 
+chromiumos-sdk/src/overlays/baseboard-octopus in 
+chromiumos-sdk/src/overlays/baseboard-octopus/profiles/base/parent is set:
+chipset-glk:base
+
+Then:
+chromiumos-sdk/src/overlays/chipset-glk/profiles/base/make.defaults
+CHROMEOS_KERNEL_SPLITCONFIG="chromeos-intel-pineview"
+
+* **block_device** device name of the persistent storage of Chromebook. 
+In some devices, this may be eMMC (mmcblkN) or NVMe, as in this case,
+in some exceptional cases it can be set to detect (like on "grunt",
+where block device name is not persistent)
+* **cros_flash_nfs** points to the debos filesystem that flashes the Chromebook.
+* **cros_flash_kernel** points to the upstream kernel that will be used for 
+the above system (it must support the peripherals of the flashed Chromebook, 
+especially persistent storage, such as eMMC, NVMe, etc)
+* **cros_image points** to **cros_flash_nfs.rootfs** repacked together with rootfs
+.bin image we built
+* **reference_kernel** is ChromeOS downstream kernel that is used for -fixed tests,
+in some cases this is the kernel provided by Google, and in some cases 
+it is extracted from the rootfs we built.
+
+And also a snippet with the tests you want to, run in test configs section:
+```
+  - device_type: asus-C436FA-Flip-hatch_chromeos
+    test_plans:
+      - cros-boot
+      - cros-boot-fixed
+      - cros-tast
+      - cros-tast-fixed
+```
+
+# How to Prepare a known type of Chromebook for KernelCI testing
+
+If you got a new Chromebook, connected it to LAVA, and want to set it up, 
+so it can be used in KernelCI, you need to flash the Chromebook with 
+the firmware(rootfs) compiled and set in kernelci configuration files.
+
+The easiest way is to generate a LAVA template for the flashing job. 
+If you look in test-configs-chromeos.yaml you will see test definition 
+"cros-flash".
+To generate a flashing template for "octopus", you need to add **cros-flash** 
+to our local configuration **test-configs-chromeos.yaml**. For example like this:
+```
+  - device_type: hp-x360-12b-ca0010nr-n4020-octopus_chromeos
+    test_plans:
+      - cros-flash
+```
+Then you need to generate, using any suitable kernel, LAVA job template:
+
+```
+./kci_test generate --lab-config=lab-collabora --install-path=_install_\
+ --output=jobs --db-config=staging.kernelci.org\
+ --storage=https://storage.chromeos.kernelci.org/\
+  --plan=cros-flash
+```
+Firmware loading can take quite a long time, 10-15 minutes, and successful 
```

Comment:
What do you mean by firmware loading here? Do you mean flashing the user-space on the eMMC?

The bootloader (Coreboot + Depthcharge) is what "firmware" usually means in the case of Chromebooks.

Reply:
Changed to "Installing OS image"